ui/packages/consul-ui/app/components/informed-action/layout.scss

```diff
@@ -2,6 +2,9 @@
   & {
     min-width: 190px;
   }
+  &.documentation {
```

Comment:
I tried making the width longer but it broke the delete confirmation. I used class 'documentation' since it could be used in the future again for "Documentation" links.

Reply:
Removed 😄